benchmark/synthetic/config.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
reference to tf/benchmark

Same for all other source files borrowed. Put exact link to the source.

Reply:
Please place exact reference links to TF benchmark source files. Same across all files bollow.